feature-dapp-impl/src/main/java/io/novafoundation/nova/feature_dapp_impl/utils/tabs/RealBrowserTabService.kt

```diff
@@ -82,12 +81,14 @@ class RealBrowserTabService(
             metaId = accountRepository.getSelectedMetaAccount().id,
             pageSnapshot = PageSnapshot.fromName(Urls.domainOf(url)),
             currentUrl = url,
-            knownDAppMetadata = getKnownDappMetadata(url),
+            knownDAppMetadata = null,
```

Comment:
I don't think we should do this just to call `syncKnownDAppMetadata`. This approach lacks readability as it splits the tab initialization into two steps